ManagedNativeInterop/ManagedNativeInterop/ManagedNativeInterop.csproj

```diff
@@ -76,6 +78,7 @@
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
     <RuntimeIdentifier>win10-arm64</RuntimeIdentifier>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```

Comment:
nit: windows arm64 does not support coreclr, so we need to set UseDotNetNativeToolchain to true here